packages/terra-paginator/src/_paginationUtils.jsx

```diff
@@ -0,0 +1,38 @@
+const calculatePages = (totalCount, itemCountPerPage) => Math.ceil(totalCount / itemCountPerPage);
+
+const pageSet = (index, totalPages) => {
+  const sequence = [];
+  const numberShiftPoint = 7;
+  const maxPagesDisplayed = 10;
+  const previousSequenceCount = 5;
+  const nextSequenceCount = 4;
+  let mutableIndex = index;
+
+  if (mutableIndex < 1) {
+    mutableIndex = 1;
+  } else if (mutableIndex > totalPages) {
+    mutableIndex = totalPages;
+  }
+
+  if (mutableIndex < numberShiftPoint) {
+    for (let i = 1; i <= maxPagesDisplayed; i += 1) {
+      sequence.push(i);
+    }
+  } else if (mutableIndex <= totalPages && mutableIndex > totalPages - maxPagesDisplayed && mutableIndex > totalPages - previousSequenceCount) {
```

Comment:
Nitpick.

For this if statement, can we do a new line between each conditional check?

Reply:
Can you provide a lint rule or style guide reference for this?